src/RazorSdk/Tool/Client.cs

```diff
@@ -169,7 +169,9 @@ public override async Task WaitForDisconnectAsync(CancellationToken cancellation
                     try
                     {
                         ServerLogger.Log($"Before poking pipe {Identifier}.");
+#pragma warning disable CA2022 // Avoid inexact read
```

Comment:
Do we know why this warning is not documented?

Reply:
It's likely new, and hyperlink doesn't resolve to a valid page. This warning started showing last week when we picked up .NET 9.0 Preview 4 SDK, in our VMR builds.